### ISSUE TITLE
release: v20.1.0-alpha20191209

### ISSUE THREAD

**Issue by danhhz** [cockroachdb/cockroach#42911]
Candidate SHA: 1da69d917105a0280aad10e86a7ee8eb2059cc92.
Deployment status: [previous](http://13.58.191.238:26258/) [candidate](http://18.219.178.251:26258/).
Nightly Suite: [ReleaseQualification](https://teamcity.cockroachdb.com/viewLog.html?buildId=1623269&buildTypeId=Cockroach_ReleaseQualification) [NightlySuite](https://teamcity.cockroachdb.com/viewLog.html?buildId=1623287&buildTypeId=Cockroach_Nightlies_NightlySuite).
 
[Release process checklist](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process)
 
Prep date: 2019-12-03
- [x] [Pick a SHA](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-PickingaSHA), fill in `Candidate SHA` above, notify #release-announce of SHA.
- [x] [Tag the provisional SHA](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-TagtheprovisionalSHA)
- [x] [Publish provisional binaries](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Publishtheprovisionalbinaries)
- [x] [Check binaries](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Checkbinaries)
- [x] [Deploy to test clusters](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-DeploytoTestClusters)
- [x] [Verify node crash reports](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Verifynodecrashreportsappearinsentry.io)
- [x] [Start release qualification suite](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [x] [Start nightly suite](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [x] fill in `Deployment status` above with clusters and `Nightly Suite` with the link to Nightly TeamCity Job
 
- Keep an eye on clusters until release date. Do not proceed below until the release date.
 
Release date: 2019-12-09
- [ ] [Check cluster status](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Checkclusterstatus)
- [ ] [Tag release](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Tagtherelease)
- [ ] [Bless provisional binaries](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Blesstheprovisionalbinaries)
 
- For production or stable releases in the latest major release series
  - [ ] Update the [Homebrew Formula](https://github.com/cockroachdb/homebrew-tap/blob/master/Formula/cockroach.rb)
  - [ ] Update the image tag in our [Orchestrator configurations](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Orchestratorconfigurations)
  - [ ] Announce version to the [registration cluster](https://github.com/cockroachlabs/production#publishing-a-new-release-version)
  - [ ] Upgrade the registration cluster (reg runs latest stable version)
 
- [ ] Update docs
- [ ] External communications for release

**Comment by danhhz** [comment]
We're going to have to pick a new candidate sha since this one is affected by #43020

**Comment by danhhz** [comment]
New date for this is 20191211. New sha is d90885c00bf2fb457121d50c554f995a13f3d7da.

**Comment by danhhz** [comment]
Closing in favor of #43056
